Assets/BowAndArrowImproved/Scripts/TeleportManager.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TeleportManager : MonoBehaviour
+{
+    [SerializeField] private GameObject playerContainer;
+    [SerializeField] private List<TeleportAnchor> anchorsList;
+
+    private Coroutine _teleportCoroutine;
+    
+    private void OnEnable()
+    {
+        _teleportCoroutine = StartCoroutine(TeleportCoroutine());
+    }
+
+    private void OnDisable()
+    {
+        if (_teleportCoroutine != null)
+        {
+            StopCoroutine(_teleportCoroutine);
+        }
+    }
+
+    private IEnumerator TeleportCoroutine()
+    {
+        while (true)
+        {
+            yield return new WaitForSeconds(10.0f);
+
+            EnableRandomAnchor();
+            yield return new WaitForSeconds(10.0f);
+
+            DisableEnabledAnchor();
```

Comment:
make the timing of the wait a serialized field so it will be easy to change it if we want to